### PR TITLE
[JustPaste.it] New ruleset

### PR DESCRIPTION
**Pull request by donced**

### CODE REVIEW

File: src/chrome/content/rules/JustPaste.it.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<ruleset name="JustPaste.it">
+	<target host=    "justpaste.it" />
+	<target host="www.justpaste.it" />
+
+  	<securecookie host="^justpaste\.it$" name=".*" />
+
+	<rule from="^http://www\.justpaste\.it/"
+		to="https://justpaste.it/" />
```
Comment (2d1) [inline src/chrome/content/rules/JustPaste.it.xml]: Why redirect to ^justpaste.it? www.justpaste.it seems to work fine. If there's no problem, please don't preempt the server's redirect. Doing so only gives an adversary on the wire greater confidence that the user is using https-everywhere.

File: src/chrome/content/rules/JustPaste.it.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+		<test url="http://justpaste.it/" />
```
Comment (2d1) [inline src/chrome/content/rules/JustPaste.it.xml]: This test is redundant, as literal targets generate implicit tests.

**Comment by 2d1** [comment]
Merged regardless for tomorrow's release.

File: src/chrome/content/rules/JustPaste.it.xml (hunk 1 above)
Comment (donced) [inline src/chrome/content/rules/JustPaste.it.xml]: I didn't consider such risk. I add it to save server's redirection.

**Pull request merged**